File: mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;
    //input number
    while (true)
    {
        height = get_int("Height: ");
        if (height <= 8 && height >= 1)
        {
            break;
        }
    }
    //output block number:8
    if (height == 8)
    {
        printf("       #  #\n      ##  ##\n     ###  ###\n    ####  ####\n   #####  #####\n  ######  ######\n #######  #######\n########  ########\n");
    }
    //output block number:7
    else if (height == 7)
    {
        printf("      #  #\n     ##  ##\n    ###  ###\n   ####  ####\n  #####  #####\n ######  ######\n#######  #######\n");
    }
    //output block number:6
    else if (height == 6)
    {
        printf("     #  #\n    ##  ##\n   ###  ###\n  ####  ####\n #####  #####\n######  ######\n");
    }
    //output block number:5
    else if (height == 5)
    {
        printf("    #  #\n   ##  ##\n  ###  ###\n ####  ####\n#####  #####\n");
    }
    //output block number:4
    else if (height == 4)
    {
        printf("   #  #\n  ##  ##\n ###  ###\n####  ####\n");
    }
    //output block number:3
    else if (height == 3)
    {
        printf("  #  #\n ##  ##\n###  ###\n");
    }
    //output block number:2
    else if (height == 2)
    {
        printf(" #  #\n##  ##\n");
    }
    //output block number:1
    else
    {
        printf("#  #\n");
    }
}

File: mario.py

def main():
    height = -1
    # input number
    while (True):
        try:
            height = int(input("Height: "))
        except:
            continue
        if (height <= 8 and height >= 1):
            break

    # output block number:8
    if (height == 8):
        print("       #  #\n      ##  ##\n     ###  ###\n    ####  ####\n   #####  #####\n  ######  ######\n #######  #######\n########  ########")
    # output block number:7
    elif (height == 7):
        print("      #  #\n     ##  ##\n    ###  ###\n   ####  ####\n  #####  #####\n ######  ######\n#######  #######")
    # output block number:6
    elif (height == 6):
        print("     #  #\n    ##  ##\n   ###  ###\n  ####  ####\n #####  #####\n######  ######")
    # output block number:5
    elif (height == 5):
        print("    #  #\n   ##  ##\n  ###  ###\n ####  ####\n#####  #####")
    # output block number:4
    elif (height == 4):
        print("   #  #\n  ##  ##\n ###  ###\n####  ####")
    # output block number:3
    elif (height == 3):
        print("  #  #\n ##  ##\n###  ###")
    # output block number:2
    elif (height == 2):
        print(" #  #\n##  ##")
    # output block number:1
    else:
        print("#  #")


if __name__ == "__main__":
    main()